app/src/ui/changes/changes-list.tsx

```diff
@@ -534,6 +534,33 @@ export class ChangesList extends React.Component<
         })
       })
 
+    if (paths.length > 1) {
+      const includeFilesOptions: IMenuItem[] = [
+        { type: 'separator' },
+        {
+          label: __DARWIN__
+            ? 'Include Selected Files'
+            : 'Include selected files',
+          action: () => {
+            selectedFiles.map(file =>
+              this.props.onIncludeChanged(file.path, true)
+            )
+          },
+        },
+        {
+          label: __DARWIN__
+            ? 'Exclude Selected Files'
+            : 'Exclude selected files',
+          action: () => {
+            selectedFiles.map(file =>
+              this.props.onIncludeChanged(file.path, false)
+            )
+          },
+        },
+      ]
+      items.push(...includeFilesOptions)
+    }
```

Comment:
Just a nit, but I don't believe we gain anything by declaring the variable and then pushing it; rather than just going ahead and pushing the items. :) 

Note.. below suggestion will likely break the linter.

```suggestion
    if (paths.length > 1) {
    items.push(
        { type: 'separator' },
        {
          label: __DARWIN__
            ? 'Include Selected Files'
            : 'Include selected files',
          action: () => {
            selectedFiles.map(file =>
              this.props.onIncludeChanged(file.path, true)
            )
          },
        },
        {
          label: __DARWIN__
            ? 'Exclude Selected Files'
            : 'Exclude selected files',
          action: () => {
            selectedFiles.map(file =>
              this.props.onIncludeChanged(file.path, false)
            )
          },
        },
      )
    }
```